django_quickbooks/urls.py

```diff
@@ -4,13 +4,14 @@
 
 from django_quickbooks.views import QuickBooksService, Support
 
+app_name = "django_quickbooks"
```

Comment:
I believe we do not need to add `app_name`.

Reply:
we don't need it, but it is nice to have it, to avoid confusion when referring to any url.
i.e. when accessing the url from the user project, in case they have similar url confs